drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/commands/ScenarioSimulationEventHandler.java

```diff
@@ -382,7 +382,7 @@ public void onEvent(SetInstanceHeaderEvent event) {
         if (context.getModel().getSelectedColumn() == null) {
```

Comment:
This check is now duplicated in `public boolean isSameInstanceType(String className)` which is invoked immediately below.

Reply:
Done!